index.bs

```diff
@@ -797,6 +797,9 @@ algorithm collects these in one place.
 <div algorithm="handle funky elements">
 To <dfn>handle funky elements</dfn> on a given |element|, run these steps:
 
+  1. If |element| is a [=shadow host=]:
+    1. Run the steps of [=sanitize a document fragment=] algorithm on
+       |element|'s [=Element/shadow root=].
```

Comment:
I suggest adding a note here that this is okay to do since we know our only entry point is the fragment parser.

If we ever have another entry point, such as an arbitrary node, we need a more complicated handshake to not violate encapsulation.

Reply:
Also, should this be part of "handle funky elements"? It doesn't matter much but it seems this applies to all elements. But maybe "handle funky element" should be "sanitize an element" and all element-logic should move here?

Reply:
If you call into `sanitizer a document fragment`, you need to pass a `Sanitizer` instance (I think `this`).
I too think it might make sense to throw stuff into a "handle element" function. Otherwise looks good.